spec/system/create_user_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Creating an Account', type: :system, driver: :selenium_chrome, js: true do
```

Comment:
Change to rack_test for github workflows to pass 😃 
```suggestion
RSpec.describe 'Creating an Account', type: :system do
  before do
    driven_by(:rack_test)
  end
```

Reply:
Worked on this. Thanks Anna!